layers/+source-control/version-control/funcs.el

```diff
@@ -59,7 +59,7 @@
   (let ((current-prefix-arg t))
     (call-interactively
      (cl-case version-control-diff-tool
-       (diff-hl     'diff-hl-margin-minor-mode)
+       (diff-hl     'diff-hl-margin-mode)
```

Comment:
Is it the only place where it need to be changed? Because the lines under looks pretty similar.